jupyter/utils/utils.py

```diff
@@ -215,3 +215,7 @@ def plot_highlight_regions(x, y, hue, hue_thresh=0, xlabel='', ylabel='',
     sns.despine()
 
     return fig, axes
+
+
+def get_epochs_info(epochs, events, event_id):
```

Comment:
Let's maybe change the way notebooks are stored in the repo and point them to the same utils file as used in the app